homeassistant/components/honeywell/climate.py

```diff
@@ -104,44 +105,36 @@ def setup_platform(hass, config, add_entities, discovery_info=None):
     username = config.get(CONF_USERNAME)
     password = config.get(CONF_PASSWORD)
 
-    if config.get(CONF_REGION) == "us":
-        try:
-            client = somecomfort.SomeComfort(username, password)
-        except somecomfort.AuthError:
-            _LOGGER.error("Failed to login to honeywell account %s", username)
-            return
-        except somecomfort.SomeComfortError:
-            _LOGGER.error(
-                "Failed to initialize the Honeywell client: "
-                "Check your configuration (username, password), "
-                "or maybe you have exceeded the API rate limit?"
-            )
-            return
-
-        dev_id = config.get("thermostat")
-        loc_id = config.get("location")
-        cool_away_temp = config.get(CONF_COOL_AWAY_TEMPERATURE)
-        heat_away_temp = config.get(CONF_HEAT_AWAY_TEMPERATURE)
-
-        add_entities(
-            [
-                HoneywellUSThermostat(
-                    client, device, cool_away_temp, heat_away_temp, username, password
-                )
-                for location in client.locations_by_id.values()
-                for device in location.devices_by_id.values()
-                if (
-                    (not loc_id or location.locationid == loc_id)
-                    and (not dev_id or device.deviceid == dev_id)
-                )
-            ]
+    try:
+        client = somecomfort.SomeComfort(username, password)
+    except somecomfort.AuthError:
+        _LOGGER.error("Failed to login to honeywell account %s", username)
+        return
+    except somecomfort.SomeComfortError:
+        _LOGGER.error(
+            "Failed to initialize the Honeywell client: "
+            "Check your configuration (username, password), "
+            "or maybe you have exceeded the API rate limit?"
         )
         return
 
-    _LOGGER.warning(
-        "The honeywell component has been deprecated for EU (i.e. non-US) "
-        "systems. For EU-based systems, use the evohome component, "
-        "see: https://www.home-assistant.io/integrations/evohome"
+    dev_id = config.get("thermostat")
+    loc_id = config.get("location")
```

Comment:
These options are not in the config schema and not documented. We should fix that.

Reply:
Added these to the schema and documentation in https://github.com/home-assistant/home-assistant.io/pull/13645